designer/client/src/containers/hooks/useSearchQuery.tsx

```diff
@@ -1,30 +1,69 @@
 import { isEqual, omitBy, uniq, without } from "lodash";
 import type { ParsedQuery, ParseOptions } from "query-string";
-import { parse } from "query-string";
-import { stringify } from "query-string";
+import { parse, stringify } from "query-string";
 import { useCallback, useEffect, useMemo, useState } from "react";
 import { useLocation } from "react-router-dom";
 
 import { ensureArray } from "../../common/arrayUtils";
 
-type QueryRecord = ParsedQuery<string | boolean | number>;
+type QueryParamValue = string | boolean | number;
+
+type QueryRecord = ParsedQuery<QueryParamValue>;
 
 const DEFAULT_ARRAY_FORMAT: ParseOptions["arrayFormat"] = "comma";
 
 function parseQuery(searchString = window.location.search): QueryRecord {
-    return parse(searchString, {
+    const parsedRawQuery = parse(searchString, {
         arrayFormat: DEFAULT_ARRAY_FORMAT,
         parseNumbers: true,
         parseBooleans: true,
-        decode: true,
+        decode: false,
```

Comment:
Do you know what the problem is with the query-string library and automatic decode?